Adding new Option UntrackedEyeFollowTracked

jensnt opened the pull request:
The new option UntrackedEyeFollowTracked will make an untracked eye follow the other one if that is tracked.

jensnt committed:
Adding new Option UntrackedEyeFollowTracked

jensnt committed:
Adding new Option UntrackedEyeFollowTracked

Chickenbreadlp commented:
It looks pretty good.
My only concern is that this PR mingles left and right pupil sizes. While I understand why it was done this way, the problem is that VRCFT internally keeps track of largest and smallest measured pupil size to calculate a normalized pupil diameter size.
Since the left and right pupil can naturally have differing pupil sizes, this could cause problems with the normalized pupil size values.

jensnt committed:
Do not set pupil size from other eye

Do not set pupil size from other eye.
See discussion in:
https://github.com/Chickenbreadlp/VRCFTVarjoModule/pull/3

jensnt commented:
> It looks pretty good. My only concern is that this PR mingles left and right pupil sizes. While I understand why it was done this way, the problem is that VRCFT internally keeps track of largest and smallest measured pupil size to calculate a normalized pupil diameter size. Since the left and right pupil can naturally have differing pupil sizes, this could cause problems with the normalized pupil size values.

OK, Fixed :)

Chickenbreadlp reviewed (approved):
LGTM 👍
